### PR TITLE
Newcomers_Guide.rst: Rephrase note

### LINKED ISSUE #183: Newcomers_Guide.rst: Rephrase note
> ```
> .. note::
> 
>     Reviewing code helps you by watching other people's mistakes and not making
>     them yourself in the future!
> ```

Imo it should also contain something in the direction of "you are improving yourself doing this" :+1:

### PR DESCRIPTION
**Pull request by Akki356**
This rephrases the note
"Reviewing code helps you by watching other people's mistakes and not making
them yourself in the future!"
by adding "you are helping yourself by doing this".

Closes #183

### CODE REVIEW

File: Developers/Newcomers_Guide.rst [hunk 1]
```diff
@@ -168,8 +168,8 @@ For more information about reviewing code, check out
 
 .. note::
 
-    Reviewing code helps you by watching other people's mistakes and not making
-    them yourself in the future!
+    Reviewing code helps you by watching other people's mistakes, you are
+improving yourself by doing this and not making them yourself in the future!
```
Comment (sils) [inline Developers/Newcomers_Guide.rst]: missing indentation here, this is not aligned with the line above

Comment (sils) [inline Developers/Newcomers_Guide.rst]: also this new sentence isn't grammatically correct anymore I think

**Comment by sils** [comment]
commit message looks ok

**Comment by Akki356** [comment]
Any suggestions @sils :)

**Comment by aptrishu** [comment]
@Akki356 Please make the changes as @sils suggested.
Indentation mismatch and grammatical mistake is there.

**Comment by aptrishu** [comment]
Still on this ? @Akki356

**Comment by Akki356** [comment]
Yes

**Comment by Nosferatul** [comment]
@Akki356 resolve conflicts from the file `Newcomers_Guide.rst`

**Comment by sils** [comment]
The File has moved to the coala repository

**Comment by Mixih** [comment]
File has been moved. You will need a new pull request.

**Pull request closed without merging**